packages/uui-css/README.md

```diff
@@ -0,0 +1,43 @@
+# uui-css
```

Comment:
This readme doesn't look correct. We need to update it to match how to load in the CSS or remove the code examples for now.